src/test/resources/datasets/twitterneed/tweet_need_test.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TweetsCollection>
+<CollectionName>Nikit's Collection</CollectionName>
+<Tweets>
+<Tweet>
+<TweetId>100000000001</TweetId>
+<TweetText>John Oliver is the best anchor ever. #lwn</TweetText>
+<TweetGroup>0</TweetGroup>
+<Mentions>
+<Mention>
+<Text>John Oliver</Text>
+<StartIndx>0</StartIndx>
+<Entity>http://dbpedia.org/resource/John_Oliver</Entity>
+</Mention>
+</Mentions>
+</Tweet>
+<Tweet>
+<TweetId>100000000002</TweetId>
+<TweetText>Gerbil is the best tool ever. #qabenchmarking</TweetText>
+<TweetGroup>0</TweetGroup>
+<Mentions>
+<Mention>
+<Text>Gerbil</Text>
+<StartIndx>0</StartIndx>
```

Comment:
In both tests, the entity has the position 0. Wouldn't it be better to have different positions to make sure that the documents are testing more situations...? :thinking: